Thalia Pay for renewals (#1226)
svthalia/concrexit: website/registrations/templates/registrations/email/renewal_accepted.txt

Unfortunately there is still one little thing that we have to take care of before
you get access to everything Thalia has to offer once again: paying your membership fees.

The membership fees are €{{ fees }}. Those can be paid in a few ways:

1. You can let us withdraw the money from your bank account via Thalia Pay. For this, you
will need to have a bank account added to your account and have signed a direct debit mandate.

> To pay with Thalia Pay, go to https://thalia.nu/user/membership/.

Reviewer: You should be able to use the base url instead of using the thalia.nu domain itself.

2. You can transfer the amount of €{{ fees }} to NL 23 INGB 0006 2341 84 with the description "Contribution {{ name }}".
Once your payment is processed by us, you will receive a confirmation email.

> 3. You can pay by card or cash 'in real life' by coming to the Thalia board room, during the lunch break (12:15-13:15)
> in M1.0.08.

Reviewer: ```suggestion
3. You can pay using debit card or cash by visiting the Thalia board room (M1.0.08) during the lunch break (12:15-13:15).
```
